File: SDKs/Aspose.Tasks_Cloud_SDK_for_Java/src/main/java/com/aspose/tasks/model/TaskItemResponse.java
package com.aspose.tasks.model;

import com.aspose.tasks.model.TaskItem;
public class TaskItemResponse {
  private TaskItem TaskItem = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getTaskItem
	 * Gets TaskItem
	 * @return TaskItem
	 */
  public TaskItem getTaskItem() {
    return TaskItem;
  }

	/**
	 * setTaskItem
	 * Sets TaskItem
	 * @param TaskItem TaskItem
	 */
  public void setTaskItem(TaskItem TaskItem) {
    this.TaskItem = TaskItem;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TaskItemResponse {\n");
    sb.append("  TaskItem: ").append(TaskItem).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Tasks_Cloud_SDK_for_CSharp/src/Com/Aspose/Tasks/Model/TaskItemResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Tasks.Model {
  public class TaskItemResponse {
    public TaskItem TaskItem { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TaskItemResponse {\n");
      sb.Append("  TaskItem: ").Append(TaskItem).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
